File: src/gate-info/src/GATEINFO.Core/Models/Gate.cs
namespace GATEINFO.Core.Models;

public class Gate
{
    public string Airport { get; set; } = string.Empty;
    public string GateNumber { get; set; } = string.Empty;
    public string Terminal { get; set; } = string.Empty;
    public string Status { get; set; } = string.Empty;
    public string AssignedFlight { get; set; } = string.Empty;
    public string Airline { get; set; } = string.Empty;
    public string LastUpdated { get; set; } = string.Empty;
}

File: src/aviation-mcp/src/models/gate.py
from pydantic import BaseModel, ConfigDict
from pydantic.alias_generators import to_camel


class Gate(BaseModel):
    model_config = ConfigDict(populate_by_name=True, alias_generator=to_camel)

    airport: str
    gate_number: str
    terminal: str
    status: str
    assigned_flight: str
    airline: str
    last_updated: str
